kitsune/users/migrations/0006_auto__add_field_profile_mozillians.py

```diff
@@ -1,22 +1,23 @@
 # -*- coding: utf-8 -*-
 import datetime
 from south.db import db
-from south.v2 import DataMigration
```

Comment:
Do not edit migrations.

Migrations should never be migrated. You need to add a new migration, not change an old one. Changing migrations will cause huge problems. For example, the users 0006 migration has already been run in production. When this code goes out, the new migration will not be run.

Do no edit migrations.